### PR TITLE
Rebalance holy staffs 

### PR DESCRIPTION
**Pull request by Mooshimi**
## About The Pull Request
-makes holy staffs belt-slottable + adds sprite for belt
-changes block from 50 -> 55
-force 5 -> 2

## Why It's Good For The Game
-gremlins steal holy staff if you hold it by pushing you over, and it involves juggling your bag around or abandoning backpack if you want it to not be stolen

## Changelog

:cl:
balance: Chaplain's holy staffs can now fit in belt slots, but also do a bit less damage and block a tiny bit more
/:cl:

### CODE REVIEW

**Comment by tralezab** [comment]
meh

**Comment by Mooshimi** [comment]
> meh

no u

File: code/game/objects/items/holy_weapons.dm [hunk 1]
```diff
@@ -241,10 +241,10 @@
...
 	righthand_file = 'icons/mob/inhands/weapons/staves_righthand.dmi'
-	w_class = WEIGHT_CLASS_HUGE
-	force = 5
-	slot_flags = ITEM_SLOT_BACK
-	block_chance = 50
+	w_class = WEIGHT_CLASS_BULKY
+	force = 2
+	slot_flags = ITEM_SLOT_BELT|ITEM_SLOT_BACK
+	block_chance = 55
 	menu_description = "A red staff which provides a high chance of blocking incoming attacks via a protective red aura around it's user, but deals very low amount of damage. Can be worn only on the back."
```
Comment (JohnFulpWillard) [inline code/game/objects/items/holy_weapons.dm:248]:
```suggestion
	menu_description = "A red staff which provides a high chance of blocking incoming attacks via a protective red aura around it's user, but deals very low amount of damage. Can be worn on the back and belt."
```

**Comment by Fikou** [comment]
meh

**Comment by necromanceranne** [comment]
meh

(also the title is deceptive as the staff isn't especially unique for pacifists and you haven't done anything to change that, so you should really provide a more accurate title. Also, if you're adding it to the belt slot, you need to make sprites for it or else it will pink sprite)

**Comment by Livrah** [comment]
meh

**Comment by SpaceVampire** [comment]
meh

**Comment by Mooshimi** [comment]
> meh
> 
> (also the title is deceptive as the staff isn't especially unique for pacifists and you haven't done anything to change that, so you should really provide a more accurate title which describes what you're actually doing. Also, if you're adding it to the belt slot, you need to make sprites for it or else it will pink sprite)

It may have been worded awkwardly, but it was saying rebalance because it's the only null rod pacifists can use

**Comment by D4C-420** [comment]
> It may have been worded awkwardly, but it was saying rebalance because it's the only null rod pacifists can use

correct me if I'm wrong but isn't pacifism supposed to be a downside? Also your reason given and pr title don't really relate to each other ( easy to steal null rod vs its the only one which pacifists "benefit " but not really as the block chance is universal.)

**Comment by AMonkeyThatCodes** [comment]
who puts a staff in their belt lmao

**Comment by Mooshimi** [comment]
> > It may have been worded awkwardly, but it was saying rebalance because it's the only null rod pacifists can use
> 
> correct me if I'm wrong but isn't pacifism supposed to be a downside? Also your reason given and pr title don't really relate to each other ( easy to steal null rod vs its the only one which pacifists "benefit " but not really as the block chance is universal.)

I didn't say they benefit, I said it's literally the only null rod that they 'can' use. All the others are weapons.

**Comment by Mooshimi** [comment]
> who puts a staff in their belt lmao

image
I think if a bulky ratvar spear can fit in belt, sure.

**Comment by Mooshimi** [comment]
image
image
image
image
same but blue
image

**Comment by ATH1909** [comment]
IMO, fikou's PR was a better way to buff holy staves: https://github.com/tgstation/tgstation/pull/58301

**Comment by Mooshimi** [comment]
> IMO, fikou's PR was a better way to buff holy staves: https://github.com/tgstation/tgstation/pull/58301

This isn't a buff, it's a balance tweak based on it being the only chap null rod not damage focused.

**Comment by optimumtact** [comment]
if your shitpost has less content than an emojii you will suffer.

**Comment by Mooshimi** [comment]
> if your shitpost has less content than an emojii you will suffer.

is it all the mehs

**Comment by TheBonded** [comment]
instead of having it lower force, you could rework it to heal people/do brain damage like the bible does. could fit the theme of cleric the staffs seem to be going for

**Comment by Mooshimi** [comment]
> instead of having it lower force, you could rework it to heal people/do brain damage like the bible does. could fit the theme of cleric the staffs seem to be going for

Then this would be an all out buff not what the pr is going for. Also the bible exists for that already.

**Comment by Mooshimi** [comment]
Oh shit I have an idea... What if I made it so the blue staff worked like the mining healer staff item and the red one is the shield. One for fighting and one for support?

**Comment by TheBonded** [comment]
> 
> 
> Oh shit I have an idea... What if I made it so the blue staff worked like the mining healer staff item and the red one is the shield. One for fighting and one for support?

making it identical to the mining healer staff would be pretty strong, but maybe making it work like a slightly worse medical beamgun that heals slightly less and has a dark blue beam could be cool. the red staff could have no block but have an inhand use that causes you to stand still and slowly take stamina/oxyloss damage (0.5-1 per tick) while creating an invincible 3x3 shield around you, and all damage dealt to the shield is cut in half and applied to you as stamina damage, so you can't do it forever but you can do it to protect your crew.

**Comment by ExcessiveUseOfCobblestone** [comment]
Really odd the focus is on pacifists when the only nerfs to the items are ones pacifists won’t ever see. Making it less viable to others is actually a buff in itself.

Probably the better solution would be to make new rods or perhaps redo others to be pacifism friendly. The idea staves are on your belt seem weird, very much so when they are almost the same height as you lol

**Comment by wesoda25** [comment]
I think the fact that you have to be holding it to reap its benefits is an interesting balance dynamic, I'd be sad to see it made more generic.

**Pull request closed without merging**